### ISSUE TITLE
Extension no longer works after update - mssql.runquery not found

### ISSUE THREAD

**Issue by ImTheWolfman** [microsoft/vscode-mssql#17720]
- MSSQL Extension Version: 1.19.1
- VSCode Version: 1.70.0
- OS Version: Windows 10 Enterprise

Steps to Reproduce:

1. Open previously saved query and attempt to run it
2. Receive error "mssql.runquery not found"

I managed to determine that this broke with MSSQL version 1.18.0. This version (and subsequent versions) are completely broken for me. In this version, I can't run queries, I can't see data sources that I had previously connected to, I can't establish new connections, etc. This behavior persists for me across all subsequent versions, including the current 1.19.1. 

If I downgrade to version 1.17.1, the extension functions again completely. I found one thing that fixes it for me, if I modify the "activationEvents" section in the "package.json" file to essentially what it was in 1.17.1, that will allow it to work again. 

To be clear, here is the section of code as it exists in 1.19.1:
`	"activationEvents": [
		"onCommand:mssql.loadCompletionExtension"
	],
`

And here is the "working" version from 1.17.1:
`	"activationEvents": [
		"onView:objectExplorer",
		"onLanguage:sql",
		"onCommand:mssql.connect",
		"onCommand:mssql.runQuery",
		"onCommand:mssql.runCurrentStatement",
		"onCommand:mssql.disconnect",
		"onCommand:mssql.manageProfiles",
		"onCommand:mssql.chooseDatabase",
		"onCommand:mssql.cancelQuery",
		"onCommand:mssql.showGettingStarted",
		"onCommand:mssql.newQuery",
		"onCommand:mssql.rebuildIntelliSenseCache",
		"onCommand:mssql.addObjectExplorer",
		"onCommand:mssql.objectExplorerNewQuery",
		"onCommand:mssql.toggleSqlCmd",
		"onCommand:mssql.loadCompletionExtension",
		"onCommand:mssql.removeAadAccount"
	],
`

**Comment by Charles-Gagnon** [comment]
@cheenamalhotra Looks like your PR here broke this : https://github.com/microsoft/vscode-mssql/pull/17562

Do you recall why you removed those events? I recall seeing a warning in VS Code about the events not being needed anymore, was that why you did that?

@ImTheWolfman If you update your VS Code version do you still see this issue?

**Comment by cheenamalhotra** [comment]
Yes I removed them due to warnings. Updating VS code should resolve the error I suppose, let me know if we should add them back and I can work on a PR.

**Comment by Charles-Gagnon** [comment]
The linting was added here : https://github.com/microsoft/vscode/issues/170984

And here's the change they mentioned (added in 1.74) : https://code.visualstudio.com/updates/v1_74#_implicit-activation-events-for-declared-extension-contributions

So that is almost assuredly the issue here. 

The min engine edition required by the extension should have been updated in that PR to prevent issues like this from causing confusion. It's been updated to 1.78 recently so we should be good going forward at least. 

I'm going to close this out for now, @ImTheWolfman please let us know if updating to the latest VS Code version doesn't solve this.

**Comment by Charles-Gagnon** [comment]
@cheenamalhotra Keeping them out is fine (that's what's recommended). The problem though at this point is we released 19.1 with a lower required engine version (^1.57)

https://github.com/microsoft/vscode-mssql/blob/release/1.19/package.json#L24

So unless we do another hotfix, then anyone running a lower VS Code version is going to see a similar error until the next version is released.

**Comment by Charles-Gagnon** [comment]
Actually, given what I just added above I'm going to reopen this and let you/ @kburtram decide what you want to do here.

**Comment by ImTheWolfman** [comment]
@Charles-Gagnon @cheenamalhotra thank you, updating VSCode worked! Once I updated it, I installed the current version of MSSQL again, now everything is working. 

I had no idea VSCode wasn't updating, but upon researching further I realized I had the User Installer version rather than the System Installer version. I know it was updating at some point in the past, but I have no idea when that ceased.
